omnibus/files/private-chef-cookbooks/private-chef/recipes/opscode-solr4.rb

```diff
@@ -123,7 +125,12 @@
 node.default['private_chef']['opscode-solr4']['command'] =  "java -Xmx#{solr_mem}M -Xms#{solr_mem}M"
 node.default['private_chef']['opscode-solr4']['command'] << "#{java_opts}"
 # Enable GC Logging (very useful for debugging issues)
-node.default['private_chef']['opscode-solr4']['command'] << " -Xloggc:#{File.join(solr_log_dir, "gclog.log")} -verbose:gc -XX:+PrintHeapAtGC -XX:+PrintGCTimeStamps -XX:+PrintGCDetails -XX:+PrintGCApplicationStoppedTime -XX:+PrintGCApplicationConcurrentTime -XX:+PrintTenuringDistribution"
+if ppc64? || ppc64le?
```

Comment:
What do you think about a refactor that would capture just the thing that is changing.  Perhaps a function like:

``
def verbose_gclog_java_opt
if ppc64? || ppc64le
   "-Xloggc:"
else
   "-Xverbosegclog:"
end
```

That way if we want to change other parts of those options, we don't have to work as hard to keep them in sync.